clients/src/main/resources/common/message/StopReplicaRequest.json

```diff
@@ -19,7 +19,11 @@
   "name": "StopReplicaRequest",
```

Comment:
👀 During the implementation, I have realized that the schema which has been proposed in the KIP does not work (my mistake). I propose to use the following schema instead. This schema will allow us to add more fields in the future. If the reviewer agree with this, I will update the KIP and and the VOTE thread.

Reply:
Yes, it makes sense to me. This is what I had assumed was in the KIP, but I guess I didn't look carefully enough.

Reply:
Perfect, thanks. I will update the KIP accordingly.